game-service/src/main/java/org/libertybikes/game/round/service/CORSFilter.java

```diff
@@ -0,0 +1,28 @@
+/**
+ *
+ */
+package org.libertybikes.game.round.service;
+
+import java.io.IOException;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerResponseContext;
+import javax.ws.rs.container.ContainerResponseFilter;
+import javax.ws.rs.ext.Provider;
+
+/**
+ * @author aguibert
+ *
+ */
+@Provider
+public class CORSFilter implements ContainerResponseFilter {
+	@Override
+	public void filter(ContainerRequestContext requestContext, ContainerResponseContext responseContext)
```

Comment:
Do we still need the entry in server.xml as well?

Reply:
I think we could do either (I did confirm that just this class without the server.xml entry fixes the CORS issue)